services/extra_classes.txt

```diff
@@ -0,0 +1 @@
+services_autorun
```

Comment:
Why did you choose to put this in a different file instead of inside of def.json?